Tasks/VsTest/vstest.ts

```diff
@@ -1396,7 +1395,10 @@ function getLatestVSTestConsolePathFromRegistry(): Q.Promise<[number, string]> {
 }
 
 function getVSTestConsole15Path(): Q.Promise<string> {
-    var xml = shelljs.exec('powershell.exe -file ' +  path.join(__dirname, './vs15Helper.ps1'), {silent:true}).output;
+    let powershellTool = tl.tool('powershell');
+    let powershellArgs = ['-file', path.join(__dirname, 'vs15Helper.ps1')]
+    powershellTool.arg(powershellArgs);
+    let xml = powershellTool.execSync().stdout;
```

Comment:
be aware that execSync will not get live output and it will buffer the output.  Convenient for running quick clis.

Consider await tool exec.